i18next-scanner.config.js

```diff
@@ -0,0 +1,48 @@
+const { i18nextToPo } = require('i18next-conv');
+const VirtualFile = require('vinyl');
+
+module.exports = {
+  input: [
+    'src/**/*.{html,js}',
+  ],
+  options: {
+    contextSeparator: '|',
+    pluralSeparator: '|',
+    ns: [
+      'categories',
+      'footer',
+      'meal-planner',
+      'navigation',
+      'search',
+      'shopping-list',
+      'starred-recipes',
+    ],
```

Comment:
Perhaps these namespaces could be sourced from the same [list](https://github.com/openculinary/frontend/blob/26cfb43779fc2ccd7cd3955f148a3c5cef6a760a/src/app/i18n.js#L23-L29) defined in the application?

Reply:
Decision: keep these separate for now, to isolate the localization extraction code from the application code